packages/lib/passport-interface/src/genericIssuanceTypes.ts

```diff
@@ -119,6 +119,39 @@ const LemonadePipelineTicketTypeConfigSchema = z.object({
   name: z.string()
 });
 
+const SemaphoreGroupConfigSchema = z.object({
+  /**
+   * Defines the set of event ID/product ID pairs that qualify a ticket-holder
+   * for membership in this group. If no product ID is specified, then all
+   * tickets for the event will qualify for group membership.
+   *
+   * The groupId is a UUID which the administrator should generate.
```

Comment:
worth specifying in this comment whether it should be globally unique, or unique on a per-pipeline basis